coretex/cli/config.py

```diff
@@ -0,0 +1,193 @@
+
+    saveConfig(config)
+
+    click.echo("User successfuly configured")
+
+
+def generateDockerScript(image: str) -> None:
+    nodeSuffix = "coretex-node-dev-gpu" if image == "gpu" else "coretex-node-dev"
+
+    dockerScriptContent = f"""#!/bin/bash
+    set -e pipefail
+    # coretex script
+
+    docker-compose -f {CORETEX_CONFIG_DIR / "docker-compose.yml"} down
+    docker pull {SERVER_URL}:{NODE_PORT}/{nodeSuffix}:latest
+    docker-compose -f {CORETEX_CONFIG_DIR / "docker-compose.yml"} up -d
+    docker ps | grep {SERVER_URL}:{NODE_PORT}/{nodeSuffix}:latest > {CORETEX_CONFIG_DIR / "container-name.log"}
+    """
+
+    dockerScript = CORETEX_CONFIG_DIR / "docker.sh"
+    with open(dockerScript, "w") as dockerFile:
+        dockerFile.write(dockerScriptContent)
+
+    os.chmod(dockerScript, 0o755)
+
+
+def generateDockerCompose(
+    name: str,
+    memory: int,
+    swap: int,
+    shm: int,
+    organizationId: str
+) -> None:
+
+    dockerComposeContent = f"""version: "3.8"
+services:
+  coretex-node-service:
+    image: {SERVER_URL}:{NODE_PORT}/coretex-node-dev:latest
+    volumes:
+      - {Path.home()}/.coretex:/root/.coretex
+    environment:
+      - CTX_NODE_NAME={name}
+      - CTX_API_URL=https://devext.biomechservices.com:29007/
+      - CTX_STORAGE_PATH=/root/.coretex
+      - CTX_ORGANIZATION_ID={organizationId}
+    deploy:
+      replicas: 1
+      update_config:
+        order: start-first
+    ports:
+      - 21000:21000
+    extra_hosts:
+      - dev.biomechservices.com:192.168.135.10
+    restart: always
+    mem_limit: {memory}g
+    memswap_limit: {swap}g
+    shm_size: {shm}gb
+    cap_add:
+      - SYS_PTRACE
+networks:
+  default:
+    name: coretex_node
+    """
+
+    dockerCompose = CORETEX_CONFIG_DIR / "docker-compose.yml"
+    with open(dockerCompose, "w") as dockerFile:
+        dockerFile.write(dockerComposeContent)
+
+
+def configNode() -> None:
+    config = loadConfig()
+    if not isUserConfigured(config):
+        click.echo("User not configured. Run \'coretex config --user\'", err = True)
+        return
+
+    if isNodeConfigured(config):
+        headers = ["node name", "server", "storage path", "node image"]
+
+        click.echo(click.style("Current Configuration:", fg = "blue"))
+        click.echo(tabulate([[
+            config.get("nodeName"),
+            config.get("serverUrl"),
+            config.get("storagePath"),
+            config.get("image")
+        ]], headers = headers))
+
+        if not click.prompt(
+            "Node configuration already exists. Would you like to update (Y/n)?",
+            type = bool,
+            default = True,
+            show_default = False
+        ):
+            return
+
+    click.echo("[Setting up coretex environment]")
+    click.echo("[Node Configuration]")
+
+    nodeName = click.prompt("Machine name", type = str)
+    organizationId = click.prompt("Organization ID", type = str)
```

Comment:
Organization ID is deprecated. Discuss the new flow for node registration with backend team.